.github/workflows/build-docker.yml

```diff
@@ -48,28 +48,13 @@ jobs:
     runs-on: ${{ inputs.RUNNER_LABEL }}
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        ref: ${{ inputs.COMMIT_HASH }}
```

Comment:
Have we removed the commit ref here? Will this stop us from checking out particular commits in the consuming repo?